### ISSUE TITLE
gradients while training

### ISSUE THREAD

**Issue by explorer359** [jacobgil/pytorch-grad-cam#40]
I have trained object detection with requires_grad=False, is this the reason i am getting none  while doing backpopagation ?

**Comment by jacobgil** [comment]
Probably, yes.
At what stage are you doing this -after the CAM was already created?
